components/match2/commons/match_summary_base.lua

```diff
@@ -838,4 +838,33 @@ function MatchSummary.defaultMapVetoDisplay(match, mapVeto)
 	return mapVeto
 end
 
+---@param games table[]
+---@param maxNumberOfBans integer
+---@param defaultIcon string?
+---@return nil
+function MatchSummary.buildCharacterBanData(games, maxNumberOfBans, defaultIcon)
+	local matchHasBans = false
+	local gamesBans = Array.map(games, function(game)
+		local extradata = game.extradata or {}
+		local banData = {{}, {}}
+		local gameHasBans = false
+		for index = 1, maxNumberOfBans do
+			if String.isNotEmpty(extradata['team1ban' .. index]) or String.isNotEmpty(extradata['team2ban' .. index]) then
```

Comment:
fwiw the keys are (sadly) not always the same across the wikis
some use tXbY some teamXbanY
(similar for picks only that there it is even worse due to c/chamipon/p/pick/h/hero/.......)

we probably should normalize them

Reply:
all of the ones in this PR uses teamXbanY form